x/sqlbuilder/upserter/upserter.go

```diff
@@ -21,7 +21,8 @@ type Statement struct {
 	InsertValues []sqlbuilder.Marker
 	SetValues    []sqlbuilder.Marker
 
-	Returning *sql.Returning
+	Returning   *sql.Returning
+	Consistency sql.Consistency
```

Comment:
this one is not useful, given the fact the `Exec` is ran in a transaction, there is no way it can be done on a replica.